bmoreonrails/rails_tutorial
# Locally sourced assets

#### What does this PR do?
* Makes local copies of jquery and google fonts, serving them from localhost instead of making requests out to google.
#### Where should the reviewers start?
* The main _change_ is to the sources for loading jquery and google fonts.
* The rest is the addition of the font definition files.
#### How should this be manually tested?
* Open up a browser console to the network activity tab. Watch the content of the "Domain" column.
* Navigate around the site.
* Do you see any requests to google or gstatic for jquery or fonts?
#### Any background context you want to provide?
This is the most naive implementation I could think of.
#### What are the relevant tickets?
#39 
#### Screenshots?
N/A
#### Questions:
1. Do we need non-latin fonts?
1. Is https://github.com/jekyll/jekyll-assets a better solution?
